scripts/install/mylar.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Mylar Installer for Swizzin
+# Author: Public920
+
+if [[ -f /tmp/.install.lock ]]; then
+    log="/root/logs/install.log"
+else
+    log="/root/logs/swizzin.log"
+fi
+
+user=$(cut -d: -f1 < /root/.master.info)
+codename=$(lsb_release -cs)
+. /etc/swizzin/sources/functions/pyenv
+
+if [[ $codename =~ ("buster"|"focal") ]]; then
+    LIST='git python3-dev virtualenv python3-virtualenv python3-pip'
+else
+    LIST='git build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev curl libbz2-dev'
+fi
+
+apt-get -y update >>"$log" 2>&1
+for depend in $LIST; do
+    apt-get -qq -y install $depend >>"$log" 2>&1 || { echo "ERROR: APT-GET could not install a required package: ${depend}. That's probably not good..."; }
+done
+
+if [[ ! $codename =~ ("buster"|"focal") ]]; then
```

Comment:
We'd like to avoid specifying lists that will only have to keep growing and be maintained, and prefer to invert them so that we can tick off the OSs that we'll be dropping eventually.
`$codename =~ ("buster"|"focal")` should be `! $codename =~ (xenial|jessie|bionic)` or the appropriate form of this idea.